sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIOSearchIT.java

```diff
@@ -21,6 +21,7 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotEquals;
 
+import com.google.common.collect.ImmutableMap;
```

Comment:
In Beam we use a vendored version of Guava. This is what's causing the precommit failure.

You'll need to import our vendored guava (and also reorder de import alphabetically I guess)
```suggestion
import org.apache.beam.vendor.guava.v26_0_jre.com.google.common.collect.ImmutableMap;
```

Reply:
Thanks, fixed!